satpy/tests/compositor_tests/test_viirs.py

```diff
@@ -311,21 +307,21 @@ def test_reflectance_corrector_abi(self):
         self.assertEqual(res.attrs['area'], area)
         self.assertEqual(res.attrs['ancillary_variables'], [])
         data = res.values
-        self.assertLess(abs(np.mean(data) - 29.907390988422513), 1e-10)
+        self.assertLess(abs(np.nanmean(data) - 26.00760944144745), 1e-10)
```

Comment:
I this change caused by the 1e30 values in the data ?

Reply:
The calculation was logically flawed as is. Because of the lons/lats not being filtered in `get_angles` there were rayleigh corrected reflectances being returned in this test that weren't accurate and should have been NaN. Since they were included in this mean calculation, the result was wrong. I've updated these numbers so the tests pass and don't include the bad values.